src/settings_type.h

```diff
@@ -493,6 +493,7 @@ struct EconomySettings {
 	uint16 town_noise_population[3];         ///< population to base decision on noise evaluation (@see town_council_tolerance)
 	bool   allow_town_level_crossings;       ///< towns are allowed to build level crossings
 	bool   infrastructure_maintenance;       ///< enable monthly maintenance fee for owner infrastructure
+	uint8  idle_loading_ct_penalty;          ///< cargotiles penalty factor (in 1/4 steps) for vehicles loading and not receiving goods
```

Comment:
What does this setting do exactly? It isn't really clear to me from the context of this PR.

In general, I would rather avoid we add settings like this, as it is more explaining to users, while they are often not in a real position to make a call on them. Maybe we should just pick for them?